docs/adguard-for-android/solving-problems/samsungpay-with-adguard-in-south-korea.md

```diff
@@ -0,0 +1,40 @@
+---
+title: How to use Samsung Pay with AdGuard in South Korea
+sidebar_position: 14
+---
+
+In the 'Advanced settings' of AdGuard for Android app there is a 'Low Level Settings' section where you can see a list of various features, the activation of which requires users' special understanding. Among these features you can find one that is called **"pref.samsungpay.autopause.enable"** and that is developed specifically for the South Korean users. Please read more to learn what this feature does.
+
+*The article is available in both languages: [Korean](#prefsamsungpayautopauseenable기능에-대한-설명) and [English](#the-prefsamsungpayautopauseenable-feature).*
+
+## "pref.samsungpay.autopause.enable"기능에 대한 설명
+
+이 설정은 특정 문제가 있는 한국인들을 위해 개발되었습니다 — "삼성 페이" 서비스는 VPN을 실행 중인 기기에서 작동되지 않습니다. AdGuard의 경우, 로컬 VPN을 이용해서 트래픽을 필터링하며 이 모드는 기본적으로 켜져 있습니다.
+
+한국 사용자들은 "삼성 페이"로 결제해야 할 때 일일이 AdGuard를 꺼야 했습니다. 하지만 이 기능을 키면 그럴 필요가 없습니다.
+
+**이 기능은 어떻게 작동될까요?**
+
+만약 **"pref.samsungpay.autopause.enable"** 옵션이 켜져 있다면, AdGuard는 사용자가 "삼성 페이" 앱을 켠 상태일 경우 자동으로 중지됩니다. 그리고 앱이 닫혔다면, AdGuard는 다시 정상적으로 작동됩니다.
+
+어떻게 작동하는지 봅시다.
+
+![samsungpay *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/samsungpay-with-adguard-in-south-korea/en.gif) 
+
+**참고:** 이 기능은 AdGuard 설정에서 VPN 필터링 모드가 켜져 있을 때만 작동합니다 — 다른 모드를 사용한다면, AdGuard는 정상적으로 작동할 것입니다.
+
+## The "pref.samsungpay.autopause.enable" feature
```

Comment:
By default, AdGuard uses Local VPN to filter traffic. This is the reason why the South Korean users have to disable AdGuard when making payments with Samsung Pay. But fortunately, now it is possible to avoid such difficulties.

The "pref.samsungpay.autopause.enable" feature was developed specifically for the South Korean users. So, what process does it initiate? If the "pref.samsungpay.autopause.enable" option is enabled, the AdGuard app is suspended when a user opens the Samsung Pay app. But once the app is closed, AdGuard resumes operation.

Let's see how it works.

![samsungpay *mobile](https://cdn.adtidy.org/content/kb/ad_blocker/android/solving_problems/samsungpay-with-adguard-in-south-korea/en.gif)

**Note:** this feature will work only if the Local VPN filtering mode is chosen in AdGuard settings. If another mode is being used, Samsung Pay will function without any interruptions.